### PR TITLE
Submission for competition

### PR DESCRIPTION
**Pull request by OpenCoreCH**

### CODE REVIEW

File: submissions/causal_machine_learning_in_healthcare.md [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+of resolution, from reactions at the molecular level to symptoms of the
+body as a whole. Furthermore, because of the previously mentioned
+challenges with RCTs, addressing these problems by collecting more data
+is often not feasible.
+
+# Estimating Causal Treatment Effects from Observations
+
+## Framework
+
+We are in the Rubin-Neyman Potential Outcomes Framework (Rubin 2005)
```
Comment (nicodjimenez) [inline submissions/causal_machine_learning_in_healthcare.md:66]: I think this section could use more discussion.  What exactly is the Rubin-Neyman Potential Outcomes Framework? Does it just describe ATE and ITE as suitable metrics for this problem? Are there alternative frameworks for measuring causality?

File: submissions/causal_machine_learning_in_healthcare.md [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
+bootstrap resampling for that, i.e. training the explanation model on
+different subsets of the data (possibly containing duplicates) and using
+the importance scores of the runs to construct confidence intervals.
+
+As AME, CXPlain provided more accurate feature importance estimates than
+LIME and SHAP, while being model-agnostic and still computationally
+efficient. Even though the approach works with arbitrary models, the
+accuracy of the estimates does depend on the predictive model and some
+model architectures seem to be better suited for explanation models.
+
```
Comment (nicodjimenez) [inline submissions/causal_machine_learning_in_healthcare.md:274]: A conclusion and future work section, even if it's short would be great to have here.

**Review by nicodjimenez — APPROVED** [review]
Thanks for submission! Interesting stuff.  Left a few comments.  It would also be great if you could include external links if possible within the main text to make it easier for the reader to explore deeper.

**Comment by nicodjimenez** [comment]
Merging now, but keeping PR open so you can submit refinements.

**Comment by OpenCoreCH** [comment]
Thanks for your feedback! I incorporated it into the text. Could you reopen the PR such that I can submit it?

**Pull request merged**